src/blocks/card.css

```diff
@@ -0,0 +1,95 @@
+.card {
+  display: flex;
+  flex-direction: column;
```

Comment:
This style is redundant, all your elements should be blocks, and each block started from the new line by default